server/api/game/core.js

```diff
@@ -45,3 +45,11 @@ export const getRandomAdjacentAvailableCell = (gridState, currentLocation) => {
   }
   return getRandomAdjacentAvailableCell(gridState, currentLocation);
 };
+
+export const moveInDirection = ({ x, y, direction }) => {
```

Comment:
moveInDirection doesn't make sense. 'newDirection' would be better or 'newLocation'

Reply:
I have just moved the code as it is. I will refactoring it